package-lock.json

```diff
@@ -0,0 +1,4917 @@
+{
```

Comment:
We shouldn't check in this file since this is a library included by other applications.